packages/camera/camera_avfoundation/example/ios/RunnerTests/AvailableCamerasTest.m

```diff
@@ -15,8 +15,7 @@ @implementation AvailableCamerasTest
 
 - (void)testAvailableCamerasShouldReturnAllCamerasOnMultiCameraIPhone {
   CameraPlugin *camera = [[CameraPlugin alloc] initWithRegistry:nil messenger:nil];
-  XCTestExpectation *expectation =
-      [[XCTestExpectation alloc] initWithDescription:@"Result finished"];
+  XCTestExpectation *expectation = [self expectationWithDescription:@"Result finished"];
```

Comment:
The change to the utility method allows using `waitForExpectationsWithTimeout:` instead of having to list every expectation.